src/config/2020.json

```diff
@@ -220,6 +220,16 @@
       "avatar_url": "https:\/\/avatars0.githubusercontent.com\/u\/1120896?v=4&s=200",
       "github": "rviscomi",
       "twitter": "rick_viscomi"
+    },
+    "max-ostapenko": {
+      "name": "Max Ostapenko",
+      "teams": [
+        "analysts",
+        "developers"
+      ],
+      "avatar_url": "https:\/\/avatars1.githubusercontent.com\/u\/1611259?v=4&s=200",
+      "github": "max-ostapenko",
+      "twitter": "themax_o"
```

Comment:
Any chance you could move this up above Ricks? I try to keep it in first name alphabetical order to make it easier to find, avoid any thoughts of precedency and satisfy my OCD (mostly the last one to be honest!) 😀